packages/scratch-vm/src/extension-support/block-relationships.js

```diff
@@ -41,6 +36,12 @@ export const addTopBlockModifier = (util, selfID, modifierKey, value) => {
       ? util[topBlockModifiers][topBlockID][modifierKey] = entry
       : util[topBlockModifiers][topBlockID] = { [modifierKey]: entry }
     : util[topBlockModifiers] = { [topBlockID]: { [modifierKey]: entry } }
+  let block_ids = Object.keys(getBlockContainer(util)._blocks);
```

Comment:
Oh interesting! I had intended for you to use the `isTopBlockID` you implemented above here, but I guess this works too! 

I'll leave it for now since we've been testing with this, but I'm going to add a comment